lib/run-datadog.sh

```diff
@@ -64,8 +64,9 @@ start_datadog() {
 
 if [ -z $DD_API_KEY ]; then
   echo "Datadog API Key not set, not starting Datadog"
+elif [ "$DD_LOGS_ENABLED" = "true" -a -z $DD_LOGS_CONFIG_TCP_FORWARD_PORT ]; then
+  echo "TCP forward port is not set, not starting Datadog"
```

Comment:
We could have a better message here, for instance `Logs are enabled but not properly setup: missing TCP forwarding port option.

Also, I think we should still start the agent in that case, but may be wrong.

Reply:
Moved this to `redirect_logs.sh`